integration-tests/validator-tool_test.go

```diff
@@ -0,0 +1,94 @@
+package integrationtests
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	binaryName            = "configuration-validator"
+	aggregationSubcommand = "aggregation"
+)
+
+func TestMain(m *testing.M) {
+	// We rely on this simple method of finding the root of the repo to be able to run the tests, as
+	// opposed to using a more elaborate method like setting an environment variable.
+	err := os.Chdir("..")
+	if err != nil {
+		fmt.Printf("could not change dir: %v", err)
+		os.Exit(1)
+	}
+
+	os.Exit(m.Run())
+}
+
+func TestConfigurationValidatorTool(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "configuration-validator integration tests suite")
+}
+
+var testCases = []TableEntry{
+	{
+		Description: "a positive test",
```

Comment:
Since this an integration test, can we try a positive test on a more complex scenario? i.e. over the complete .yaml in the tech spec?